src/components/VirtualList/Config.ts

```diff
@@ -116,24 +116,24 @@ const istioType: ResourceType<IstioConfigItem> = {
 };
 
 export const IstioTypes = {
-  gateway: { name: 'Gateway', icon: 'G' },
-  virtualservice: { name: 'VirtualService', icon: 'VS' },
-  destinationrule: { name: 'DestinationRule', icon: 'DR' },
-  serviceentry: { name: 'ServiceEntry', icon: 'SE' },
-  rule: { name: 'Rule', icon: 'R' },
-  adapter: { name: 'Adapter', icon: 'A' },
-  template: { name: 'Template', icon: 'T' },
-  quotaspec: { name: 'QuotaSpec', icon: 'QS' },
-  quotaspecbinding: { name: 'QuotaSpecBinding', icon: 'QSB' },
-  policy: { name: 'Policy', icon: 'P' },
-  meshpolicy: { name: 'MeshPolicy', icon: 'MP' },
-  servicemeshpolicy: { name: 'ServiceMeshPolicy', icon: 'SMP' },
-  clusterrbacconfig: { name: 'ClusterRbacConfig', icon: 'CRC' },
-  rbacconfig: { name: 'RbacConfig', icon: 'RC' },
-  servicemeshrbacconfig: { name: 'ServiceMeshRbacConfig', icon: 'SRC' },
-  sidecar: { name: 'Sidecar', icon: 'S' },
-  servicerole: { name: 'ServiceRole', icon: 'SR' },
-  servicerolebinding: { name: 'ServiceRoleBinding', icon: 'SRB' }
+  gateway: { name: 'Gateway', slug: 'gateways', icon: 'G' },
```

Comment:
slug ?

Reply:
It is the term used for humanizing IDs in the URL. Probably born because of all the SEO.
https://en.wikipedia.org/wiki/Clean_URL#Slug

For example, in `http://www.contentwithstyle.co.uk/content/clean-urls-for-a-better-search-engine-ranking` the `slug` is `clean-urls-for-a-better-search-engine-ranking`.

Anyway, I changed to follow the other places in the project where we use term `url`.